install.sh

```diff
@@ -382,6 +438,7 @@ Options:
 --python, -p                python version (only python3 versions are supported)
 --with-qt                   build the Qt GUI
 --without-qt                don't build the Qt GUI
+--with-ln-messaging     build and use c-lightning for message channels
```

Comment:
nit - alphabetical order and spacing.
```suggestion
--with-ln-messaging         build and use c-lightning for message channels
--with-qt                   build the Qt GUI
--without-qt                don't build the Qt GUI
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/7a5e4c3369966dda121ac01a09dd999071c8dc4a